src/components/Transaction.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 
-const Transaction = () => {
+const Transaction = (props) => {
   return (
```

Comment:
may be nice to de-structure the transaction here - `const { transaction } = props`